api/v1alpha1/constants.go

```diff
@@ -1,8 +1,9 @@
 package v1alpha1
 
 const (
-	LabelNameTenant = "syn.tools/tenant"
-	FinalizerName   = "cluster.lieutenant.syn.tools"
+	LabelNameTenant       = "syn.tools/tenant"
+	FinalizerName         = "cluster.lieutenant.syn.tools"
+	PipelineFinalizerName = "cluster.lieutenant.syn.tools"
```

Comment:
It might not collide but i'd personally choose a different name for clarity

```suggestion
	PipelineFinalizerName = "cluster.lieutenant.syn.tools/pipelines"
```